README.md

```diff
@@ -245,3 +245,11 @@ Run below command from kernelci-api/api directory:
 pytest
 ```
 This will start running test cases from kernelci-api/api/test_api.py and display results.
+
+## Generate SSH key
+
+Use below command to generate SSH key. It will store private key to `authorized_keys` file and public key to `authorized_keys.pub` file.
+
+```
+$ ssh-keygen -f authorized_keys
+```
```

Comment:
Actually that's not really how `authorized_keys` is meant to be used: the private key should stay on the host outside of Docker, and the public key should be listed in `authorized_keys`.  For example:
```
$ ssh-keygen -f ~/.ssh/id_rsa_kernelci-api
$ cat ~/.ssh/id_rsa_kernelci-api.pub >> docker/ssh/user-data/authorized_keys
```
Then the user should be able to ssh to the running Docker container using:
```
$ ssh -i ~/.ssh/id_rsa_kernelci-api -p 8022 kernelci@localhost
Linux b57d306e7629 4.19.0-10-amd64 #1 SMP Debian 4.19.132-1 (2020-07-24) x86_64

The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
kernelci@b57d306e7629:~$ 
```
with this added change to export the container's port 22 to the host as 8022 (arbitrary number, related to adding parameters for port numbers):
```diff
diff --git a/docker/docker-compose.yaml b/docker/docker-compose.yaml
index 65d7b729fcb7..15b1f5b60b89 100644
--- a/docker/docker-compose.yaml
+++ b/docker/docker-compose.yaml
@@ -44,6 +44,8 @@ services:
     volumes:
       - './storage/data:/home/kernelci/data'
       - './ssh/user-data:/home/kernelci/.ssh'
+    ports:
+      - '8022:22'
 
 volumes:
   mongodata:
```